README.md

```diff
@@ -115,22 +122,50 @@ organizations: [
   99999999 // your org added here
 ]
 ```
+* Connect to the SQS build queue
 
-* Run the server with `npm start` (You can use `npm run watch:server` for the server to restart when you save file changes and `npm run watch:client` to rebuild front-end files on each save) at the directory of the project on your local computer.
+In development, Federalist uses the staging build queue.
+To connect to the build queue, get the credentials for the staging build queue and add them to the SQS config in your `config/local.js`
+
+```
+sqs: {
+  accessKeyId: "ABC123",
+  secretAccessKey: "456DEF",
+  region: "us-est-1",
+  queue: "https://sqs.us-east-1.amazonaws.com/789/ghi",
+}
+```
+
+* Connect to the S3 bucket
+
+In development, Federalist uses the staging S3 bucket.
+To connect to the bucket, get the credentials for the staging bucket and add them to the S3 config in your `config/local.js`.
+
+```
+s3: {
+  accessKeyId: "ABC123",
+  secretAccessKey: "456def",
+  region: "us-gov-west-1",
+  bucket: "789ghi",
```

Comment:
Okay, I added some sentences above the examples to clarify that they are example values only and a note about where to get the actual values.